README.md

```diff
@@ -16,6 +16,7 @@ Python 3.8.5+
 
 ```console
 $ pip install parse-emails
+$ brew install libmagic
```

Comment:
?

Reply:
Had some issues configuring the tests module, turns out this is a necessary step (@ilaner mentioned it's not the first time developers had related issues).